src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -146,7 +154,7 @@ class SwipeableDrawer extends React.Component {
 
     this.setState({ maybeSwiping: true });
     if (!open) {
-      this.setPosition(this.getMaxTranslate() - (disableDiscovery ? 0 : swipeAreaWidth), {
+      this.setPosition(this.getMaxTranslate() + (disableDiscovery ? 20 : -swipeAreaWidth), {
```

Comment:
What does this change do? Wouldn't this move the drawer even if disableDiscovery is `true`?

Reply:
> Hide the box-shadow when disableDiscovery is set